config/samples/example-test.sh

```diff
@@ -234,8 +235,31 @@ function getPodImage() {
 	done
 }
 
+function getHelmRevision() {
+	namespace=$1
+	releaseName=$2
+	wantRevision=$3
+	START_TIME=$(date +%s)
+	while true; do
+		get=$(helm status -n ${namespace} ${releaseName} -o json | jq '.version')
```

Comment:
**Suggestion:** Use double quotes around variable references to prevent word splitting and pathname expansion.
```suggestion
  get=$(helm status -n "${namespace}" "${releaseName}" -o json | jq '.version')
```